gprofiler/profilers/java.py

```diff
@@ -244,7 +244,8 @@ def read_output(self) -> Optional[str]:
 class JavaProfiler(ProcessProfilerBase):
     JDK_EXCLUSIONS = ["OpenJ9", "Zing"]
 
-    def __init__(self, frequency: int, duration: int, stop_event: Event, storage_dir: str):
+    def __init__(self, frequency: int, duration: int, stop_event: Event, storage_dir: str, java_mode: str):
+        self._java_mode = java_mode  # Not currently in use
```

Comment:
Add `assert` on its value to be what you expect.

And whys saving it on the instance?